src/main/scala/com/fulcrumgenomics/sv/tools/FilterAndMerge.scala

```diff
@@ -33,166 +31,106 @@ class FilterAndMerge
     val dict = SequenceDictionary(this.dict)
 
     // Read in the pileups
-    val parser = DelimitedDataParser(path=input, delimiter='\t')
-    val iter   = parser
-      .map(row => Pileup(row))
-      .filter(pileup => pileup.total >= minPre)
+    val iter   = Metric.iterator[BreakpointPileup](input)
+      .filter(_.total >= minPre)
       .toIndexedSeq
-      .groupBy(_.sameStrand) // this ensures we get pileups adjacent on the same strand
-      .flatMap { case (_, pileups) =>
-        pileups.sortBy(p => (p.leftContig, p.rightContig, p.leftPos, p.rightPos))
-      }
+      .sortBy(p => (p.left_contig, p.right_contig, p.left_strand, p.right_strand, p.left_pos, p.right_pos))
       .iterator
       .bufferBetter
 
     // Iterate
     val results = IndexedSeq.newBuilder[MergedPileup]
     while (iter.hasNext) {
-      val first   = iter.next()
-      val builder = IndexedSeq.newBuilder[Pileup]
-      builder += first
-      val pileups = nextGroup(iter=iter, builder=builder, prev=first)
+      val pileups = nextGroup(iter=iter)
       val merged  = MergedPileup(pileups)
-      if (merged.counter.total >= minPost) {
+      if (merged.total_evidence >= minPost) {
         results += merged
       }
     }
 
     // Write
-    val writer = Io.toWriter(output)
-    writer.write(MergedPileup.header.mkString("\t"))
-    writer.write('\n')
-    results.result()
-      .sortBy { p =>
-        (dict(p.left_contig).index, dict(p.right_contig).index, p.left_start, p.left_end, p.right_start, p.right_end)
-      }
-      .foreach { merged =>
-        val values = MergedPileup.values(merged)
-        writer.write(values.mkString("\t"))
-        writer.write('\n')
-      }
-    writer.close()
+    Metric.write(output, results.result().sorted)
```

Comment:
changes the sort :/

Reply:
Well the old sort involved the `sameStrand` which I don't think makes too much sense any more.  If you have a different preference for how to order fields I'm game - I thought I did this in a way that would give you the closest to what you had before (it's using the ordering on the new metric class, not Breakpoint).